microprofile.ls/org.eclipse.lsp4mp.ls/src/test/java/org/eclipse/lsp4mp/services/ApplicationPropertiesDiagnosticsTest.java

```diff
@@ -746,4 +746,33 @@ public void validateMultilineKey() {
 				d(0, 0, 2, 4, "Unknown property 'qu.application.name'",
 						DiagnosticSeverity.Warning, ValidationType.unknown));
 	}
```

Comment:
Maybe add another test with
```java
quarkus.log.category.*.level=DEBUG
```
to show that this is also highlighted and to catch any regressions that change this.

Reply:
fixed